src/components/Cluster/ClusterDetail/InstanceTypeSelector/InstanceTypeRow.tsx

```diff
@@ -0,0 +1,64 @@
+import { Global } from '@emotion/core';
+import styled from '@emotion/styled';
+import { IInstanceType } from 'hooks/useInstanceTypeSelectionConfiguration';
+import PropTypes from 'prop-types';
+import React, { FC } from 'react';
+import {
+  BulletStyle,
+  CursorPointerCell,
+  Tr,
+} from 'UI/ExpandableSelector/Items';
+import RadioInput from 'UI/Inputs/RadioInput';
+
+interface IInstanceTypeRow extends IInstanceType {
+  isSelected: boolean;
+  selectInstanceType(instanceType: string): void;
+}
+
+const LeftAlignedCell = styled(CursorPointerCell)`
+  text-align: left;
+`;
+
+const InstanceTypeRow: FC<IInstanceTypeRow> = ({
+  cpu,
+  ram,
+  name,
+  description,
+  isSelected,
+  selectInstanceType,
+}) => {
+  return (
+    <>
+      <Global styles={BulletStyle} />
```

Comment:
Btw, a question about this. Do the styles have to be global? We could have conflicting classNames across components

Would it work to style the `Tr` component and add the bullet styling there?

For example:

```javascript
const StyledTr = styled(Tr)`
	.release-selection-radio {
		...
	}
`;
```

Reply:
I think you're totally right! The current implementation will also inject the styles n times for n items so not really ideal.

I'll change that